src/ui/ConsoleView.js

```diff
@@ -159,14 +159,14 @@ class ConsoleView extends Component {
               </button>
             </div>
             {this.state.method !== 'get' ? (
-              <div class="form-inline">
-                <label for="body" class="sr-only">
+              <div className="form-inline">
+                <label for="body" className="sr-only">
```

Comment:
Thought this should be caught by flow

Reply:
Good point. I’m not sure whether the built-in div component set that restriction in its props type definition. If it does, then the problem might be how I import it into this file. I’ll take a look.